ee/clickhouse/timer.py

```diff
@@ -56,7 +56,10 @@ def schedule(self, callback: Callable, *args, **kwargs) -> TimerTask:
 
     def cancel(self, task: TimerTask) -> None:
         with self.lock:
-            del self.tasks[task.id]
+            try:
+                del self.tasks[task.id]
```

Comment:
Thoughts about checking if the key exists or not here instead?

I don't see anything else going wrong here, and if it does I don't want a bare `except` catching it?

Reply:
Wanted to be even more race-aware, e.g. task completes in-between checking the key and deleting :)